oap-server/oal-grammar/src/main/antlr4/org/apache/skywalking/oal/rt/grammar/OALLexer.g4

```diff
@@ -61,6 +61,26 @@ SRC_PROFILE_TASK: 'profile_task';
 SRC_PROFILE_TASK_LOG: 'profile_task_log';
 SRC_PROFILE_THREAD_SHANPSHOT: 'profile_task_segment_snapshot';
 
+// Constructors symbols
```

Comment:
@kezhenxu94, the like expression seems to have some issue when @arugal did local test. He did local tests, so he moved the symbols here.

Reply:
Please recheck. @arugal Could your current test case reproduce the error? If these changed back to the old way.

Reply:
From my understanding, antlr and all word analyzer are order sensitive. It could match the keyword or string before these matches.

Reply:
> Could your current test case reproduce the error? If these changed back to the old way.

Based on the master branch
![image](https://user-images.githubusercontent.com/26432832/91317879-29a27700-e7ed-11ea-8e71-9173a9e07222.png)

Reply:
@kezhenxu94 From my reading of the old OALLexer definition, `like` or `in` could be tagged as `IDENTIFIER`. So when the Parser wants to match the expression, mismatch happens. I don't know whether this is the case, or there is deeper reason for word analysis.

```
expression
    : booleanMatch | stringMatch | greaterMatch | lessMatch | greaterEqualMatch | lessEqualMatch | notEqualMatch | booleanNotEqualMatch | likeMatch
    ;

booleanMatch
    :  conditionAttribute DUALEQUALS booleanConditionValue
    ;
```

Reply:
`latency like` could be `IDENTIFIER` + `IDENTIFIER`. That is why when we talked about the language, there is a concept called KEYWORDs, which are not allowed to be used as an IDENTIFIER name, right?

Reply:
Yes confirmed, moving the `like` exactly on top of `IDENTIFIER` solves the problem, it WAS treated as an identifier